test-service/rebar.config

```diff
@@ -17,8 +17,19 @@
     {apps, [ldclient, ts_app]}
 ]}.
 
-{profiles, [{prod, [{relx, [{dev_mode, false}
-    ,{include_erts, true}]}]}]}.
+{profiles, [
+    {prod, [{relx, [{dev_mode, false}
+    ,{include_erts, true}]}]},
+    {otp26,
```

Comment:
Has to be conditionally added with a profile, because for older OTP versions it isn't recognized.